### PR TITLE
MSI.AppPlayer version 1.0

### PR DESCRIPTION
**Pull request by Twig6943**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/102515)

### CODE REVIEW

File: manifests/m/MSI/AppPlayer/1.0/MSI.AppPlayer.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: MSI.AppPlayer
+PackageVersion: 1.0
+Installers:
+- Architecture: x86
+  InstallerType: zip
```
Comment (BrandonWanHuanSheng) [inline manifests/m/MSI/AppPlayer/1.0/MSI.AppPlayer.installer.yaml:8]: It look like this one is an portable.

Comment (BrandonWanHuanSheng) [inline manifests/m/MSI/AppPlayer/1.0/MSI.AppPlayer.installer.yaml:8]: It look like this one is an portable

File: manifests/m/MSI/AppPlayer/1.0/MSI.AppPlayer.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: MSI.AppPlayer
+PackageVersion: 1.0
+Installers:
+- Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: exe
+  NestedInstallerFiles:
+  - RelativeFilePath: MSI APP Player.exe
```
Comment (BrandonWanHuanSheng) [inline manifests/m/MSI/AppPlayer/1.0/MSI.AppPlayer.installer.yaml:11]: It look like there is a  PortableCommandAlias after the line 11

**Comment by BrandonWanHuanSheng** [comment]
It look like inside zip file is just the real installer..

**Comment by BrandonWanHuanSheng** [comment]
This is normally related to BlueStack Installer. If Bluestack Installer gives me the commands, we can know how to install.

**Comment by stephengillie** [comment]
Hi @Twig6943,

The installation isn't unattended:
InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230414123114

Is there an installer switch to bypass this and have it install automatically?

**Comment by BrandonWanHuanSheng** [comment]
It look like the offline installer can work silently. Only online installer did not work due to they restart the setup.

**Comment by BrandonWanHuanSheng** [comment]
> Hi @Twig6943,
> 
> The installation isn't unattended: InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230414123114
> 
> Is there an installer switch to bypass this and have it install automatically?

No, only offline installer have silent parameters -s due to online installer restart the setup.

**Pull request closed without merging**